Src/CustomVisionCompanion/CustomVisionCompanion/Views/SettingsPage.xaml

```diff
@@ -27,13 +27,16 @@
     <Grid Style="{StaticResource MasterGridStyle}">
         <ScrollView Style="{StaticResource ContentGridStyle}">
             <StackLayout>
-                <Label Style="{StaticResource CaptionLabelStyle}" Text="Prediction Key" />
+                <Label
+                    Margin="0,0,0,10"
```

Comment:
This new margin are OK even for Android and UWP?

Reply:
yes of course